wadm/lib/wadm/deployments/deployment_monitor.ex

```diff
@@ -19,6 +19,8 @@ defmodule Wadm.Deployments.DeploymentMonitor do
 
   @decay_timer_ms 39_000
 
+  @configuredcache [Wadm.Observer.RedisCache]
```

Comment:
replace with Wadm.Observer.MnesiacCache one could also send to both redis and mnesia for comparison.